src/lib389/lib389/instance/setup.py

```diff
@@ -446,6 +449,36 @@ def create_from_cli(self):
             slapd['root_password'] = rootpw1
             break
 
+        # Database implementation (db_lib)
+        while 1:
```

Comment:
while True.

Reply:
I looked at other occurrences as I cut and pasted that piece of code from previous option handling
and indeed the file is plagued by that one.
grep -c 'while 1' instance/setup.py
11
I will global substitute them ...